index.html

```diff
@@ -29,7 +29,9 @@ <h2>Environments</h2>
 
 		<h2>Release Calendar</h2>
 
-		<p><i>* Disclaimer: Out of all test releases, only release deployed to Kovan network contains all necessary configuration required for full testing.</i></p>
+		<p><i>* Disclaimer 1: Out of all test releases, only release deployed to Kovan network contains all necessary configuration required for full testing.</i></p>
+		<p><i>* Disclaimer 2: Releases that contain <b>active</b> in the URL are those that have been deployed, executed and in use.</i></p>
+		<p><i>* Disclaimer 3: Releases that contain <b>number</b> (e.g. 1.1.0) in the URL might be deployed, but not yet in use.</i></p>
```

Comment:
I think it should say:
"Disclaimer 3: Releases that contain <b>number</b> (e.g. 1.1.0) in the URL are deployed, but might not be in use yet."